lib/DDG/Spice/Clojars.pm

```diff
@@ -0,0 +1,43 @@
+package DDG::Spice::Clojars;
+# ABSTRACT: This spide searches for an artifact on Clojars.
+
+
+use DDG::Spice;
+
+# Caching - https://duck.co/duckduckhack/spice_advanced_backend#caching-api-responses
```

Comment:
You can also remove this comment and any others that don't explain this specific code.